targets/win32/build-binary

```diff
@@ -87,7 +87,7 @@ if [ `is_standalone_app` = "yes" ]; then
 else
   if [ `is_gui_app` = yes ]; then
     veval "$SYS_CC -I$SYS_PREFIX/include \
-      $cflag_additions -mwindows win32_microgl.c main.c icon.o -o $tgt \
+      $cflag_additions -mwindows -fpermissive win32_microgl.c main.c icon.o -o $tgt \
```

Comment:
Shouldn't the `-fpermissive` also only be applied when using a c++ compiler, as I thought this was only needed if you use non-conforming code?